### PR TITLE
Add translations for offline map

### PR DESCRIPTION
**Pull request by jorgkv**
Adds translations for offline map. Only adds norwegian, nynorsk and english. English is fallback, so should be ok.

### CODE REVIEW

**Comment by jorgkv** [comment]
@widforss feel free to add sv translations if you want to

**Comment by widforss** [comment]
Swedish added. I think I got it right, but since "karta" and "paket" have different genders in Swedish, the gender of "Nedladdat"/"Nedladdad" and "Producerat"/"Producerad" are dependent on which of those nouns the user think they are bound to. I chose "Nedladdat" and "Producerat" as I would say they are used with "paket" in this case.

File: src/assets/i18n/sv.json [hunk 1]
```diff
@@ -183,7 +183,30 @@
     "OBSERVATIONS": "Observationer"
   },
   "OFFLINE_MAP": {
-    "TITLE": "Offline kartor"
+    "OFFLINE_MAP_PAGE_TITLE": "Offline kartor"
+  },
```
Comment (jorgkv) [inline src/assets/i18n/sv.json:187]: Er tekstene delt opp i to objekter her? Kan være det fører til at det ene eller andre objektet forsvinner, evt at jsonen blir ugyldig.

**Review by gruble — APPROVED** [review]
Bra jobba!
Var noen småting jeg oppdaget under testing, men tror jeg fikk luket dem ut. Du kan jo titte på det jeg har sjekket inn.

**Pull request merged**